packages/node/src/edge-functions/edge-handler.ts

```diff
@@ -169,7 +169,7 @@ export async function createEdgeEventHandler(
       const fakeStackTrace = `    at (${entrypointRelativePath})`;
       const urlPath = extractUrlPath(entrypointRelativePath);
```

Comment:
Should `extractUrlPath()` return with the forward slash instead of modifying the log?

Reply:
Yes, that better represents what `urlPath` is: [`6b362d9` (#9428)](https://github.com/vercel/vercel/pull/9428/commits/6b362d9ee9f62ed37dbf7a04c216002d52bf6365)